pages/common/regex.md

```diff
@@ -0,0 +1,37 @@
+# regex
+
+> Regular expressions (`regex`) are patterns used to match, search, and manipulate text.
+> Not a direct command, but syntax to be used with other commands.
+> More information: <https://cheatography.com/davechild/cheat-sheets/regular-expressions/>.
+
+- Match any single character:
+
+`.`
+
+- Match the start of a line:
+
+`^{{hello}}`
+
+- Match zero or more repeated characters:
+
+`{{a}}*`
+
+- Match an optional character:
+
+`colou?r`
```

Comment:
```suggestion
```
I'm limiting the examples to what grep supports without `--extended-regexp` flag